hack/loadtest/test-crp.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: placement.kubernetes-fleet.io/v1beta1
```

Comment:
Hi Britania! I noticed that the YAMLs still do not have a rollout strategy part; if you would like to add them separately I am also OK. The timeout could have an effect on the results.